src/main/java/org/myrobotlab/service/RemoteSpeech.java

```diff
@@ -0,0 +1,77 @@
+package org.myrobotlab.service;
+
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.myrobotlab.io.FileIO;
+import org.myrobotlab.logging.Level;
+import org.myrobotlab.logging.LoggerFactory;
+import org.myrobotlab.logging.LoggingFactory;
+import org.myrobotlab.service.abstracts.AbstractSpeechSynthesis;
+import org.myrobotlab.service.config.HttpClientConfig;
+import org.myrobotlab.service.config.RemoteSpeechConfig;
+import org.myrobotlab.service.data.AudioData;
+import org.slf4j.Logger;
+
+public class RemoteSpeech extends AbstractSpeechSynthesis<RemoteSpeechConfig> {
+
+  private static final long serialVersionUID = 1L;
+
+  public final static Logger log = LoggerFactory.getLogger(RemoteSpeech.class);
+
+  public transient HttpClient<HttpClientConfig> http = null;

```

Comment:
yeah .. .that has always been my first reaction ... 
but should it ?
Interestingly, this would work with 
mouth.http.someMethod
vs the "safe" constructed references of mouth_http.somMethod
:thinking: so i started thinking .. what would be the downside of making this reference to a service public ?